src/core/config.py

```diff
@@ -88,6 +88,6 @@ def get_bool(setting: str) -> bool:
 
 # URL nenaprasno
 URL_SITE = "https://ask.nenaprasno.ru/"
-URL_SITE_DONATION = "https://ask.nenaprasno.ru/#donation"
 FORM_URL_FUTURE_EXPERT = "https://forms.gle/DGMUm35bxZytE3QLA"
 URL_SERVICE_RULES = "https://docs.google.com/document/d/1hW2HUv9aWQMnUBuIE_YQEtmIDDbk8KhpychckbyaIEQ/edit"
+URL_ASK_NENAPRASNO = "https://ask-nnyp.klbrtest.ru/"
```

Comment:
Я бы  значение переменной использовал без последней косой черты
```suggestion
URL_ASK_NENAPRASNO = "https://ask-nnyp.klbrtest.ru"
```
А использовал потом таким образом
```
f"({config.URL_ASK_NENAPRASNO}/consultation/redirect/{consultation_id})"
```
```
InlineKeyboardButton("Поддержать проект", url=f"{config.URL_SITE}/#donation"),
```
На мой взгляд проще воспринимать.

Reply:
во[т как раз при](https://github.com/Studio-Yandex-Practicum/ask_nenaprasno_bot/pull/120/files)мер того, что у нас используются разные переменные и нет порядка в них . Тут используется совсем другая переменная URL_SITE. В чём между ними разница? Давайте использовать какую-то одну.
Мне больше нравится новый вариант, как более читаемый, а первый предлагаю удалить. Плюс, в этом же ПРе пройдись по всем ссылкам и поправь хардкоды. И кстати, значение переменной нужно делать не константой, а брать из .env файла (писал уже об этом)